bmaptools/BmapCopy.py

```diff
@@ -173,6 +173,8 @@ def __init__(self, image, dest, bmap=None, image_size=None):
         self._progress_format = None
         self.set_progress_indicator(None, None)
 
+        self.psplash_pipe = None
```

Comment:
I think the idea n this code is to prefix every private class attribute with "_". This attribute looks private too. 

Also a nitpick. This is about progress, so the attribute basically belongs to the "self.*progress*" group of attributes, therefore can be put close to them without an extra newline.

Reply:
Makes sense. I'll address it.